test/gpu/native/studies/chop/chplGPU.skipif

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Skip if not doing performance testing
+if [ -z "$CHPL_TEST_PERF" ]; then
+  echo "True"
+  exit
+fi
+
+# Skip this test if we cannot connect to github, which is required to
+# run the test.
+ping -c 1 github.com &> /dev/null
+status=$?
+if ! [ $status -eq 0 ]; then
+  echo "True"
+  exit 1
```

Comment:
I don't think we want `exit 1` here. That will cause sub_test to raise an error because the skipif failed to run correctly, but in this case we just want to skip and nothing really failed